src/keymap.cpp

```diff
@@ -268,86 +268,86 @@ static const std::vector<OpDesc> opdescs = {
 		"set-filter",
 		"F",
 		translatable("Set a filter"),
-		KM_FEEDLIST | KM_ARTICLELIST
+		KM_FEEDLIST | KM_ARTICLELIST | KM_SEARCHRESULTSLIST
 	},
 	{
 		OP_SELECTFILTER,
 		"select-filter",
 		"f",
 		translatable("Select a predefined filter"),
-		KM_FEEDLIST | KM_ARTICLELIST
+		KM_FEEDLIST | KM_ARTICLELIST | KM_SEARCHRESULTSLIST
 	},
 	{
 		OP_CLEARFILTER,
 		"clear-filter",
 		"^F",
 		translatable("Clear currently set filter"),
-		KM_FEEDLIST | KM_HELP | KM_ARTICLELIST
+		KM_FEEDLIST | KM_HELP | KM_ARTICLELIST | KM_SEARCHRESULTSLIST
 	},
 	{
 		OP_BOOKMARK,
 		"bookmark",
 		"^B",
 		translatable("Bookmark current link/article"),
-		KM_ARTICLELIST | KM_ARTICLE | KM_URLVIEW
+		KM_ARTICLELIST | KM_SEARCHRESULTSLIST | KM_ARTICLE | KM_URLVIEW
 	},
 	{
 		OP_EDITFLAGS,
 		"edit-flags",
 		"^E",
 		translatable("Edit flags"),
-		KM_ARTICLELIST | KM_ARTICLE
+		KM_ARTICLELIST | KM_SEARCHRESULTSLIST | KM_ARTICLE
 	},
-	{OP_NEXTFEED, "next-feed", "j", translatable("Go to next feed"), KM_ARTICLELIST},
+	{OP_NEXTFEED, "next-feed", "j", translatable("Go to next feed"), KM_ARTICLELIST | KM_SEARCHRESULTSLIST},
```

Comment:
I don't think this operation makes any sense for the search dialog: the feed that the search dialog shows is ephemeral, it was created just now to display a particular subset of articles. Thus, there is no "next feed" for it.

Same goes for `next-feed`, `prev-unread-feed`, and `next-unread-feed`.